tools/windows-node-installer/pkg/cloudprovider/aws/ec2.go

```diff
@@ -150,6 +150,8 @@ func (a *AwsProvider) CreateWindowsVM() (credentials *types.Credentials, rerr er
         (New-Object -TypeName System.Net.WebClient).DownloadFile($url,  $file)
         powershell.exe -ExecutionPolicy ByPass -File $file
         powershell -NonInteractive -ExecutionPolicy Bypass Add-WindowsCapability -Online -Name OpenSSH.Server~~~~0.0.1.0
+        powershell -NonInteractive -ExecutionPolicy Bypass New-NetFirewallRule -DisplayName "OpenKubeletPort"
+        -Direction Inbound -Action Allow -Protocol TCP -LocalPort 10250 -EdgeTraversalPolicy Allow
```

Comment:
nit: You can consider moving `10250` to a constant as you use it in multiple places, and so the function of it is understood.

Reply:
You can also move the firewall name `OpenKubeletPort` to a const as you use that in multiple places as well